src/utils.ts

```diff
@@ -1,4 +1,6 @@
-import type { DeepReadonly } from './superTypes'
+import type { DeepReadonly, FileSizeUnit, ReadableFileSize } from './superTypes'
+import short from 'short-uuid'
+import type { StoreFilesPayload } from 'domain/file/command/storeFile'
```

Comment:
The `utils` file can't be coupled with the domain itself, you should depend on abstractions.